deployments/optimism/BadgerBridgeZeroController.json

```diff
@@ -0,0 +1,352 @@
+{
+  "address": "0x5556834773F7c01e11a47449D56042cDF6Df9128",
+  "abi": [
+    {
+      "inputs": [],
+      "name": "BURN_GAS_DIFF",
```

Comment:
Can we add this to `deployments.json` within this PR? The FE requires it be there. I usually just manually add it.